aspnetcore/fundamentals/routing.md

```diff
@@ -399,7 +399,9 @@ In the following example, a middleware uses the <xref:Microsoft.AspNetCore.Routi
 
 ## Route templates
 
-Tokens within `{}` define route parameters that are bound if the route is matched. More than one route parameter can be defined in a route segment, but route parameters must be separated by a literal value. For example, `{controller=Home}{action=Index}` isn't a valid route, since there's no literal value between `{controller}` and `{action}`. Route parameters must have a name and may have additional attributes specified.
+Tokens within `{}` define route parameters that are bound if the route is matched. More than one route parameter can be defined in a route segment, but route parameters must be separated by a literal value. For example:
+
+   `{controller=Home}{action=Index}` isn't a valid route, because there's no literal value between `{controller}` and `{action}`. Route parameters must have a name and may have additional attributes specified.
```

Comment:
```suggestion
   `{controller=Home}{action=Index}` 

isn't a valid route, because there's no literal value between `{controller}` and `{action}`. Route parameters must have a name and may have additional attributes specified.
```